src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -201,8 +201,8 @@ export default class TutorialView {
                 if (prop.aircraft.list.length <= 0) {
                     return t;
                 }
-
-                return t.replace('{CALLSIGN}', prop.aircraft.list[0].callsign);
+                const reverseList = [...prop.aircraft.list].reverse();
```

Comment:
I believe this does not need the spread into an array-- this is the same as `prop.aircraft.list.reverse();`. And we are trying to eliminate use of the `prop` as well.

Also, we actually would want the closest controllable arrival and first departure, while this would give us the furthest. Perhaps something like `_filter(aircraftList, (aircraft) => aircraft.flightPhase === FLIGHT_PHASE.APRON)[0]` or similar would be what we need here.